doc/pymode.txt

```diff
@@ -373,7 +373,7 @@ Set pep257 options                               *'g:pymode_lint_options_pep257'
 Set pylint options                               *'g:pymode_lint_options_pylint'*
 >
     let g:pymode_lint_options_pylint =
-        \ {'max-line-length': g:pymode_options_max_line_length})
+        \ {'max-line-length': g:pymode_options_max_line_length}
```

Comment:
I had tried to copy & paste this line from the help doc into my ftplugin for python files, but was getting syntax errors b/c of the trailing ")"